static/js/overlays.js

```diff
@@ -102,6 +102,8 @@ exports.open_modal = function (name) {
     blueslip.debug('open modal: ' + name);
 
     $("#" + name).modal("show").attr("aria-hidden", false);
+    // Disable background mouse events when modal is active
+    $('.overlay.show').attr("style","pointer-events : none");
```

Comment:
The whitespace around commas (etc.) here is not standard; generally we do space-after-comma..

Reply:
(I'm sorta surprised the linter didn't complain about this -- we should see why we don't have an  `eslint` rule for this).

Reply:
Fixed the linter issue in 838dd6d2731ed09bd253866fb61d6099f5f1bf2f

Reply:
Okay, I've fixed the whitespaces.